.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: weekly
+  target-branch: master
+  labels:
+  - dependency_updates
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: monthly
```

Let's make sure we always have the latest GH Actions versions :) And then just stay with using `vMAJOR` tags.
```suggestion
    interval: daily
```